### PR TITLE
updated ci-kafka-client 3.1.0 to amazonlinux:2

### PR DESCRIPTION
**Pull request by EloghosaU1**
-related to jira ticket 2725
-ci-kafka-client-3.1.0 update to linux:2
-image built and tested locally.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,12 +3,11 @@ FROM alpine:latest as builder
 ARG KAFKA_VERSION=3.1.0
 ARG SCALA_VERSION=2.13
 
-RUN apk update && \
```
Comment (DrewHarvard) [inline Dockerfile:6]: You have removed the apk update from the Dockerfile. 

Need to add `RUN apk update && apk add --no-cache wget tar gzip` to the Dockerfile.

File: Dockerfile [hunk 2]
```diff
@@ -17,7 +16,7 @@ ARG YUM_REPOSITORY=yum-repository.platform.aws.chdev.org
 
 ENV PATH "${PATH}:${KAFKA_HOME}/bin"
 
-COPY --from=builder /kafka_${SCALA_VERSION}-${KAFKA_VERSION} /opt/kafka_${SCALA_VERSION}-${KAFKA_VERSION}
+COPY --from=builder /opt/kafka_${SCALA_VERSION}-${KAFKA_VERSION} /opt/kafka
```
Comment (DrewHarvard) [inline Dockerfile:19]: You should make use of the variable for `/opt/kafka`

Comment (DrewHarvard) [inline Dockerfile]: Did this need to change? We need to provide a consistent approach across the 3 Dockerfiles.

Comment (EloghosaU1) [reply Dockerfile]: Noted, thank you.

File: Dockerfile (hunk 1 above)
Comment (EloghosaU1) [reply Dockerfile:6]: noted. thanks

File: Dockerfile
Comment (DrewHarvard) [inline Dockerfile:1]: Thanks for the updates, more consistent with the other Kafka version Dockerfiles.

Can you please update the following:

`ENV PATH "${PATH}:${KAFKA_HOME}/bin"` becomes `ENV PATH="${PATH}:${KAFKA_HOME}/bin"`

and 

`yum install -y java-17-amazon-corretto-headless \` becomes `yum install -y java-17-amazon-corretto-headless &&  \`

Comment (EloghosaU1) [reply Dockerfile:1]: noted, thanks

**Review by DrewHarvard — APPROVED** [review]

**Pull request merged**